presto-main/src/main/resources/webapp/src/components/QueryHeader.jsx

```diff
@@ -53,6 +53,12 @@ export class QueryHeader extends React.Component {
                             </div>
                         </div>
                     </td>
+                    <td>
+                        <a onClick={() => $.ajax({url: '/v1/query/' + query.queryId + '/preempted', type: 'PUT', data: "Preempted via web UI"})} className="btn btn-warning"
+                           target="_blank">
+                            Pre-empt
```

Comment:
Let's remove the dash, since "preempt" seems to be more common. The common trend is for words to start hyphenated, then have it removed as they get more common. For example, *e-mail* is now *email*.  Preempt seems to have moved well into the non-hyphenated phase (Merriam-Webster agrees).